packages/vue/src/stories/releases/v0.10.4/v0.10.4.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Releases/v0.10.4 - Latest/Change log" />
```

Comment:
Shouldn't this be just `Releases/v0.10.4/Change log`? (without the `latest` tag). If I understood correctly, the latest release is v0.11.0?

Reply:
we wanted to release this hotfix before 0.11.0 so it will be the latest :) After releasing 0.11.0 it will take the latest label

Reply:
Sounds good to me!